hbase-server/src/main/java/org/apache/hadoop/hbase/util/HBaseFsck.java

```diff
@@ -707,6 +709,18 @@ public int onlineConsistencyRepair() throws IOException, KeeperException, Interr
     return errors.getErrorList().size();
   }
 
+  private void reportUnknownServers() throws IOException {
+    List<ServerName> unknownServers = admin.listUnknownServers();
+    if (!unknownServers.isEmpty()) {
+      unknownServers.stream().forEach(serverName -> {
+        errors.reportError(ERROR_CODE.UNKNOWN_SERVER,
+          "Found unknown server, "
+            + " some of the regions hold by this server may not get assigned. "
```

Comment:
nit: hold -> held. 
also we could remove an extra space between "server,  some"